src/web/mjs/engine/Settings.mjs

```diff
@@ -315,4 +336,40 @@ export default class Settings {
 
         }
     }
+
+    /**
+     * HYPOFLEX
+     * Save the given value for the given key in the bookmark storage
+     */
+    saveBookmarks( key, value, indentation ) {
```

Comment:
Move method either to `Storage` or to both, `BookmarkManager` and `ChaptermarkManager`

Reply:
will change  to both files instead.

Reply:
Reason for moving  to storage instead of keeping it in  settings is to minimize app performance impact?

Reply:
No its more to honor clean code principle (class responsibility), settings shall not handle bookmark and chapter-mark I/O.